src/ggml.c

```diff
@@ -15977,6 +15981,10 @@ static thread_ret_t ggml_graph_compute_thread(void * data) {
     int node_n = -1;
 
     while (true) {
+        if (cplan->abort_callback && cplan->abort_callback(cplan->abort_callback_data)) {
+            state->shared->node_n += 1;
```

Comment:
I found an issue while testing that sometimes all but one thread aborts and that one thread is spinning, waiting for `shared->node_n` to be updated. So, I decided to modify `node_n`